components/ai_rewriter/common/mojom/ai_rewriter.mojom

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) 2024 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+module ai_rewriter.mojom;
+
+import "ui/gfx/geometry/mojom/geometry.mojom";
+
+interface AIRewriterAgent {
+  GetFocusBounds() => (gfx.mojom.RectF rect);
+};
+
+// Interface that the Rewriter page uses to communicate with the browser process.
+interface AIRewriterService {
```

Comment:
AIRewriterPageHandle might be a better name as it's implementation is not a browser context keyed service.